acceptance/tests/connect/connect_inject_test.go

```diff
@@ -242,8 +242,7 @@ const multiportAdmin = "multiport-admin"
 // two ports. This tests inbound connections to each port of the multiport app, and outbound connections from the
 // multiport app to static-server.
 func TestConnectInject_MultiportServices(t *testing.T) {
-	t.Skipf("skipping until multi-port workaround is supported")
-	for _, secure := range []bool{false, true} {
+	for _, secure := range []bool{true} {
```

Comment:
If we aren't testing the `secure=false` case anymore, we could just hardcode it and remove the loop.